bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/discovery/SatelDeviceDiscoveryService.java

```diff
@@ -15,10 +15,7 @@
 import static org.openhab.binding.satel.internal.SatelBindingConstants.*;
 
 import java.nio.charset.Charset;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
wildcards are only allowed for static imports, please revert

Reply:
Done.